File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/repository/SellerDashboardRepository.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.repository;

import com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.entity.SellerDashboardStats;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;
import java.util.UUID;

public interface SellerDashboardRepository extends JpaRepository<SellerDashboardStats, UUID> {
    Optional<SellerDashboardStats> findBySellerId(UUID sellerId);
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/seller/repository/SellerDashboardRepository.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.repository

import com.multicampus.gamesungcoding.a11ymarketserver.feature.seller.entity.SellerDashboardStats
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface SellerDashboardRepository : JpaRepository<SellerDashboardStats, UUID> {
    fun findBySellerId(sellerId: UUID): SellerDashboardStats?
}
